_posts/devops/2016-04-17-docker-getting-started.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Docker - Getting Started
```

Comment:
@harshadyeola Change title to Install Docker
Also change file name to 2016-04-17-install-docker.markdown

As we publish series of tutorials so its better we use SEO friendly name as people mostly search how to install xyz